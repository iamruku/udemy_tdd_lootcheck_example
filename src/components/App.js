import React from 'react';
import Wallet from './Wallet';
import Loot from './Loot';

export default () => (
	<div>
		<h2>Loot Check</h2>
		<hr></hr>
		<Wallet></Wallet>
		<Loot></Loot>
	</div>
);
