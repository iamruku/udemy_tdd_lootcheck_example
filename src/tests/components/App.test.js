import React from 'react';
import { shallow } from 'enzyme';
import '../setupTests';
import App from '../../components/App';

describe('App', () => {
	const wrapper = shallow(<App></App>);

	describe('initial dom content', () => {
		it('renders wrapper', () => {
			expect(wrapper).toMatchSnapshot();
		});

		it('contains a `Connect(Wallet)` component', () => {
			expect(wrapper.find('Connect(Wallet)').exists()).toBe(true);
		});

		it('contains a `Connect(Loot)` component', () => {
			expect(wrapper.find('Connect(Loot)').exists()).toBe(true);
		});
	});
});
