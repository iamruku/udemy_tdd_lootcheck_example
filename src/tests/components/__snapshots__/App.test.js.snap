// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App initial dom content renders wrapper 1`] = `ShallowWrapper {}`;
